extensions/http-curl/tests/ThreadPoolAdjust.cpp

```diff
@@ -31,9 +31,11 @@
 #include "processors/LogAttribute.h"
 #include "utils/IntegrationTestUtils.h"
 
+constexpr uint64_t INCREASED_WAITTIME_MSECS = DEFAULT_WAITTIME_MSECS * 1.5;
+
 class HttpTestHarness : public IntegrationBase {
  public:
-  HttpTestHarness() {
+  HttpTestHarness() : IntegrationBase(INCREASED_WAITTIME_MSECS) {
```

Comment:
I would hard-code the wait time value here instead of defining it as `DEFAULT_WAITTIME_MSECS * 1.5`, because I don't think we want this wait time to change if we change the default later.

Also, since `wait_time_` is only used as a parameter to `verifyLogLinePresenceInPollTime()`, ie. it will stop waiting earlier if the log line is found earlier, we could make it longer, eg. 5 seconds or even 10 seconds.

Reply:
Changed it to be hard-coded 5000 ms, instead of the previously calculated 4500 ms.